umibukela/templates/about.html

```diff
@@ -260,9 +260,9 @@ <h2 class="text-white">How do we do CBM?</h2>
           <nav class="steps-sidebar">
             <h4>The CBM Process</h4>
             <ul class="nav nav-stacked">
-              <li><a href="#government">1. Establish gov. relationship</a></li>
+              <li><a href="#government">1. Build Gov. relationship</a></li>
```

Comment:
Although I could be wrong: http://www.chicagomanualofstyle.org/qanda/data/faq/topics/CapitalizationTitles/faq0015.html